networkit/cpp/graph/TopologicalSort.cpp

```diff
@@ -47,9 +84,18 @@ void TopologicalSort::run() {
     hasRun = true;
 }
 
+node TopologicalSort::mapNode(node u) {
+    if (!nodeIdMap.empty())
```

Comment:
```suggestion
    if (!nodeIdMap.empty()) {
        auto retIt = nodeIdMap.find(u);
        return retIt->second;
    } else if (computedNodeIdMap.has_value())
```

Reply:
For some reason, I was under the impression that `at()` and `find()` have unwanted overhead like in vectors. Of course, there's no overhead, thanks for the suggestion!

Reply:
For some reason, I had thought `at()` or `find()` would cause a performance overhead, but that's clearly not the case for `unordered_map`. Now it all falls back into place.